actionpack/lib/action_dispatch/journey/route.rb

```diff
@@ -50,14 +50,15 @@ def self.verb_matcher(verb)
       end
 
       def self.build(name, app, path, constraints, required_defaults, defaults)
-        request_method_match = verb_matcher(constraints.delete(:request_method))
-        new name, app, path, constraints, required_defaults, defaults, request_method_match, 0, {}
+        new name: name, app: app, path: path, constraints: constraints,
```

Comment:
Is there any worth changing `new name, app, path, constraints, ...` to `new name: name, app: app, path: path, constraints: constraints, ...`?
The `self.build` still have the dependency of remember parameters'
positions, and the `new name: name, app: app, path: path, constraints: constraints, ...` looks redundant to me.

Reply:
Good point. We should update `self.build` to use keyword args too. We could even consider removing it, I don't see `Route.build` used anywhere other than tests unless it is used somewhere else magically. 🤔

Reply:
Good catch @kamipo! As @gmcgibbon pointed out, `Route.build` wasn't necessary. My last commit removes it. Thanks for your reviews guys ❤️